types/cosmos.go

```diff
@@ -155,52 +155,34 @@ func (tx Tx) Successful() bool {
 
 // Message represents the data of a single message
 type Message struct {
-	TxHash    string
-	Index     int
-	Type      string
-	Value     string
-	Addresses []string
-	Height    int64
+	TxHash string
+	Index  int
+	// Message type url
+	Type string
+	// Value that can be used to create a Any object to
+	// unmarshal the message
+	Value  []byte
```

Comment:
Now we provide the message bytes. The message can be unmarshaled by creating an `sdk.Any` instance and then calling the `codec.UnpackAny` function.